app/src/main/java/org/p2p/wallet/auth/repository/UserSignUpDetailsStorage.kt

```diff
@@ -24,12 +24,14 @@ class UserSignUpDetailsStorage(
         val signUpDetails: Web3AuthSignUpResponse
     )
 
-    fun save(data: Web3AuthSignUpResponse, userId: String): Boolean {
+    fun save(data: Web3AuthSignUpResponse, userId: String, isCreate: Boolean = true): Boolean {
         val key = generatePrefsKey(userId)
         val value = SignUpUserDetails(userId, data)
 
         accountStorage.saveObject(KEY_LAST_DEVICE_SHARE_ID, value)
-        accountStorage.saveString(KEY_IN_SIGN_UP_PROCESS, key)
+        if (isCreate) {
```

Comment:
isCreateFlow?
what about removing this key?

Reply:
this key remove only on finish create flow,  and remove after create wallet is finish